components/ResourceMenu.tsx

```diff
@@ -0,0 +1,57 @@
+import { fhirJson } from "@fhir-typescript/r4-core";
+import { Button, Menu } from "@mantine/core";
+import Link from "next/link";
+
+export interface MenuProps {
+  resourceType: string;
+  id: string | string[] | undefined;
+  measureArray: (fhirJson.BundleEntry | null)[] | undefined;
```

Comment:
Maybe we should rename this prop since this component is general for any resource type, not just measures?